db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -9412,7 +9450,7 @@ public ColPermsDescriptor getColumnPermissions(UUID tableUUID,
     public RoutinePermsDescriptor getRoutinePermissions(UUID routineUUID,String authorizationId) throws StandardException{
         RoutinePermsDescriptor key=new RoutinePermsDescriptor(this,authorizationId,null,routineUUID);
 
-        return (RoutinePermsDescriptor)getPermissions(key);
+        return (RoutinePermsDescriptor)getPermissions(key, true);
```

Comment:
Why not introduce an overloaded method `getPermissions` with `true` being default? Perhaps even with a different name?

Reply:
Changed the code as suggested.